flask_config.py

```diff
@@ -1,13 +1,10 @@
 """Flask configuration class."""
 import os
 
+
 class Config:
     """Base configuration variables."""
     SECRET_KEY = os.environ.get('SECRET_KEY')
     if not SECRET_KEY:
-        raise ValueError("No SECRET_KEY set for Flask application. Did you forget to run setup.sh?")
-        
-    TRELLO_BASE_URL = 'https://api.trello.com/1'
-    TRELLO_API_KEY = os.environ.get('TRELLO_API_KEY')
-    TRELLO_TOKEN = os.environ.get('TRELLO_TOKEN')
-    TRELLO_BOARD_ID = os.environ.get('TRELLO_BOARD_ID')
+        raise ValueError(
+            "No SECRET_KEY set for Flask application. Did you forget to run setup.sh?")
```

Comment:
(Minor, but worth noting we removed the setup.sh file a while back, so this might be better updated!)